README.md

```diff
@@ -311,6 +312,23 @@ Closes an active modal window; applicable only for **custom modal**.
 await sdk.execute(Command.CLOSE_MODAL);
 ```
 
+### Redirect to
+
+Redirects user to specified view.
+
+**Parameters**
+
+| Parameter  | Type          | Description                                                                                                          | Notes                                                                                                                                                                                                |
+|------------|---------------|----------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| view       | View          | View where the user will be redirected to                                                                            | required<br/><br/>Supported views:<br/>View.DEALS<br/>View.LEADS<br/>View.ORGANIZATIONS<br/>View.CONTACTS<br/>View.CAMPAIGNS<br/>View.PROJECTS<br/>View.SETTINGS - redirects to custom settings page |
```

Comment:
Question (non-blocking): Should we mention here that the app should have its own custom settings page implemented?

Reply:
I think this should be logical, but I can add link to custom settings page documentation, so it's clear what is meant by this?